### ISSUE TITLE
cannot rollback drop_view

### ISSUE THREAD

**Issue by princejoseph** [scenic-views/scenic#395]
```
class DropEditedPostsView < ActiveRecord::Migration[6.1]
  def change
    drop_view :edited_posts, revert_to_version: 2
  end
end
```

Rollback fails with this error:

```
Running via Spring preloader in process 35380
== 20230815171901 DropEditedPostsView: reverting =================
-- create_view(:edited_posts, {:version=>2})
rake aborted!
StandardError: An error has occurred, this and all later migrations canceled:

wrong number of arguments (given 2, expected 1)
/Users/prince/.rvm/gems/ruby-3.1.0/gems/scenic-1.5.2/lib/scenic/statements.rb:25:in `create_view'
```

**Rails**: 6.1.7.4
**Ruby**: 3.1.0p0

**Comment by princejoseph** [comment]
> What Ruby version are you on?On Aug 17, 2023, at 12:35 PM, Prince Joseph ***@***.***> wrote:﻿ class DropEditedPostsView < ActiveRecord::Migration[6.1] def change drop_view :edited_posts, revert_to_version: 2 end end fails with this error: Running via Spring preloader in process 35380 == 20230815171901 DropEditedPostsView: reverting ================= -- create_view(:edited_posts, {:version=>2}) rake aborted! StandardError: An error has occurred, this and all later migrations canceled: wrong number of arguments (given 2, expected 1) /Users/prince/.rvm/gems/ruby-3.1.0/gems/scenic-1.5.2/lib/scenic/statements.rb:25:in `create_view' —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

Ruby: 3.1.0p0

**Comment by derekprior** [comment]
@princejoseph from your stack trace, it appears you are (or were) running Scenic 1.5.2. I believe this bug was fixed in v1.5.5, which was released about 18 months before this issue was opened. Try updating Scenic.
